media-libs/openexr/files/openexr-3.1.5-musl-i386.patch

```diff
@@ -0,0 +1,19 @@
+--- a/src/lib/Iex/IexMathFpu.cpp
```

Comment:
Please send this upstream (and include the context from the PR message at the top of this patch).